packages/api/src/models/medical/cq-patient-data.ts

```diff
@@ -0,0 +1,30 @@
+import { DataTypes, Sequelize } from "sequelize";
+import { CQData, PatientCQData } from "../../domain/medical/cq-patient-data";
+import { BaseModel, ModelSetup } from "../_default";
+
+export class PatientCQDataModel extends BaseModel<PatientCQDataModel> implements PatientCQData {
```

Comment:
OOC, whats the reason for a separate data model for CQ data as opposed to just adding it to the data in the current Patient Data model?

Reply:
So we've discussed this with the team, and the reason for this is that we're gonna have too much CQ data to be storing on the existing patient table. We're expecting up to 20-30 CQ links per patient, with 4 attributes for each for now. In the future, we might be adding even more stuff on there. Not only that, the patient update uses a table lock, and we want to avoid scenarios with conflicting updates happening at the same time.